### PR TITLE
fix(L2): several fixes

### PR DESCRIPTION
**Pull request by ilitteri**
**Motivation**

The CLI was merged to main with some isses.

**Description**

Fixes several issues:
- `OperatorConfig`. Rename struct fields to proper ones.
- Ethereum Rust L2 CLI `stack` command. Add a `force` optional argument for `start`, `purge`, and `restart` command.
- Ethereum Rust L2 CLI `wallet` command. Gas estimation is failing, so we hardcode the `gas_limit` to the double of the default transaction cost; fixed the `transfer` transaction hash display.

### CODE REVIEW

File: cmd/ethereum_rust_l2/src/commands/stack.rs [hunk 1]
```diff
@@ -26,21 +26,31 @@ pub(crate) enum Command {
         l1: bool,
         #[clap(long, help = "Shuts down the L2 node.", default_value_t = true)]
         l2: bool,
+        #[clap(short = 'y', long, help = "Forces the restart without confirmation.")]
```
Comment (ManuelBilbao) [inline cmd/ethereum_rust_l2/src/commands/stack.rs:29]: I think you forgot to customize the message for each command (_Forces the ~~restart~~ shutdown_)

**Review by ManuelBilbao — APPROVED** [review]
Just a nit comment, but everything looks good

File: cmd/ethereum_rust/ethereum_rust.rs [hunk 2]
```diff
@@ -104,7 +104,7 @@ async fn main() {
     let data_dir = matches
         .get_one::<String>("datadir")
         .unwrap_or(&default_datadir);
-    let store = Store::new(data_dir, EngineType::Libmdbx).expect("Failed to create Store");
+    let store = Store::new(data_dir, EngineType::InMemory).expect("Failed to create Store");
```
Comment (mpaulucci) [inline cmd/ethereum_rust/ethereum_rust.rs:107]: this is a problem.

**Review by mpaulucci — COMMENTED** [review]
please provide a more descriptive description and title

File: cmd/ethereum_rust/ethereum_rust.rs (hunk 2 above)
Comment (ilitteri) [reply cmd/ethereum_rust/ethereum_rust.rs]: You're right! This shouldn't have been pushed

**Comment by ilitteri** [comment]
> please provide a more descriptive description and title

Done!

**Pull request merged**